manifests/base/deployment.yaml

```diff
@@ -34,6 +34,9 @@ spec:
         - name: https
           containerPort: 443
           protocol: TCP
+        - name: metrics
```

Comment:
Exposing unecrypted endpoint should not be part of default config

Reply:
Agree, we should have some mechanisms to secure this endpoint. We could move the default port to 8443.